dev/com.ibm.ws.install.featureUtility/resources/com/ibm/ws/install/featureUtility/internal/resources/FeatureUtilityToolOptions.nlsprops

```diff
@@ -72,6 +72,17 @@ installFeature.option-desc.--from=\
 \tUse this option to specify the file path of a Maven artifact         \n\
 \trepository as a source for featureUtility to install assets.
 #------------------------------\n at 72 chars -- leading tab-----------\n\#
+installFeature.option-key.--verify=\
+\ \ \ \ --verify=[enforce|warn|skip|all]
+installFeature.option-desc.--verify=\
+\tUse this option to specify how features should be verified. The      \n\
+\tdefault option, "enforce," verifies the signatures of all Liberty    \n\
+\tfeatures except for user features. The "warn" option is the same as  \n\
+\t"enforce," but it will not terminate installation in case of failure.\n\
```

Comment:
```suggestion
\t"enforce", but it will not terminate installation in the event that verification fails
```suggestion
\t"enforce," but it will not terminate installation in case of failure.\n\
```.\n\
```